### PR TITLE
Update Set-CsTeamsMeetingPolicy.md

### PR DESCRIPTION
**Pull request by emmaryan123**
Updating the registration policy with correct defaults

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 1]
```diff
@@ -814,16 +814,16 @@ Accept wildcard characters: False
...
 
 Possible values:
-- true
-- false
+- Everyone
+- EveryonenCompany
 
 ```yaml
-Type: Boolean
+Type: Strrng
```
Comment (Srividya-Ajay) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:821]: Spell error - This should say "String"

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 2]
```diff
@@ -814,16 +814,16 @@ Accept wildcard characters: False
 ### -AllowMeetingRegistration
 
 Possible values:
-- true
-- false
+- Everyone
+- EveryonenCompany
```
Comment (Srividya-Ajay) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:818]: EveryoneInCompany

**Review by dariomws — COMMENTED** [review]
I've tested this parameter on versions 2.0.0 and 1.1.11-preview, and it's boolean. Maybe I'm missing something.

Thanks!

**Comment by get-itips** [comment]
> I've tested this parameter on versions 2.0.0 and 1.1.11-preview, and it's boolean. Maybe I'm missing something.
> 
> Thanks!

Same here.

**Comment by tseward** [comment]
Ditto....

```
PS C:\> Set-CsTeamsMeetingPolicy -AllowMeetNow asfasdf
Cannot convert 'System.String' to the type 'System.Boolean' required by parameter 'AllowMeetNow'.
```

**Comment by yogkumgit** [comment]
@emmaryan123

**Review by emmaryan123 — COMMENTED** [review]
Updated the spelling mistakes

**Comment by dariomws** [comment]
@emmaryan123 we tested the parameter and verified that it's boolean.

**Comment by emmaryan123** [comment]
> @emmaryan123 we tested the parameter and verified that it's boolean.

@dariomws how do I update the PR with that? I am not very ffamiliar with github. Thanks

**Comment by get-itips** [comment]
> > @emmaryan123 we tested the parameter and verified that it's boolean.
> 
> @dariomws how do I update the PR with that? I am not very ffamiliar with github. Thanks

Hello @emmaryan123 how are you? hope you are doing good, is the PR necessary if `-AllowMeetingRegistration` is boolean and only accepts true or false?

image

Was `-AllowMeetingRegistration` the intended parameter to update? Thank you

**Comment by emmaryan123** [comment]
Abandoning PR and will create new updated

**Comment by emmaryan123** [comment]
Closing and abandoning, will create new PR

**Pull request closed without merging**